### ISSUE TITLE
apispec doesn't work when logging in with ui

### ISSUE THREAD

**Issue by leafty** [SwissDataScienceCenter/renga#100]
Issue with url: `http://localhost/api/swagger.json`
Logs:
```
apispec_1           | invalid request block size: 5225 (max 4096)...skip
apispec_1           | [uwsgi-http key: localhost client_addr: 172.18.0.5 client_port: 58040] hr_instance_read(): Connection reset by peer [plugins/http/http.c line 646]
```
Probable cause: server rejects the request headers due to size with cookies set by ui after login.

Reproducing:
1. Log in to `http://localhost/ui/`
2. Try to load `http://localhost/admin/swagger/` -> fails

**Comment by jirikuncar** [comment]
The `/ui` should set `Path` for cookies.

**Comment by leafty** [comment]
Ok, will push a fix in renga-ui

**Comment by leafty** [comment]
Pushed issue to https://github.com/SwissDataScienceCenter/renga-ui/issues/33

**Comment by leafty** [comment]
Solved by SwissDataScienceCenter/renga-ui#33.
Manually clearing localhost cookies from browser might required.
